### ISSUE TITLE
New Project Role column on Users Page

### ISSUE THREAD

**Issue by ferristocrat** [storj/storj#5743]
## Motivation/Why?
- We want to add more information to the "Users" table, so that project owners and members know whether each user on the project is an owner vs. a member, and which users have been invited but are not yet project members. 

## Acceptance Criteria

- The "Team" list contains a new column with the users "Role" which can be one of the following:
  - Owner
  - Member
  - Invited
- The name of an invited user is the email address without the @ and the domain. Ex. `michael` if the email is michael@storj.io (see Figma for example)
## Out of Scope
 - Other changes to the Users page/table including (these are covered here: https://github.com/storj/storj/issues/5742)
   - UI updates
   - Pagination
 - "Three dots" menu for row actions

## Figma Designs
- https://www.figma.com/file/HlmasFJNHxs2lzGerq3WYH/Satellite-GUI?node-id=19573%3A189269&t=0CbvPRNNDA121SqC-1

**Comment by mobyvb** [comment]
I think we should wait for this ticket: https://github.com/storj/storj/issues/5748
before starting on this one - the investigation might result in slightly different decisions about what columns to add and where
